test/e2e/broker_default_test.go

```diff
@@ -219,6 +222,44 @@ func TestDefaultBrokerWithManyTriggers(t *testing.T) {
 	}
 }
 
+// This test is for avoiding regressions on the trigger dependency annotation functionality.
```

Comment:
Do we have a doc link for this functionality? If yes can you add it to the comment?

Reply:
I don't think so. I can only find the PR and issue for this..., maybe I can add the corresponding issue for creating this functionality to the comment?